outscale/data_source_outscale_dhcp_option.go

```diff
@@ -63,23 +62,10 @@ func dataSourceOutscaleDHCPOption() *schema.Resource {
 
 func dataSourceOutscaleDHCPOptionRead(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*OutscaleClient).OSCAPI
-
-	filters, filtersOk := d.GetOk("filter")
-	dhcpID, dhcpIDOk := d.GetOk("dhcp_options_set_id")
-	if !dhcpIDOk && !filtersOk {
-		return fmt.Errorf("One of filters, or dhcp_options_set_id must be provided")
-	}
-
 	params := oscgo.ReadDhcpOptionsRequest{}
-	if dhcpIDOk {
-		params.Filters = &oscgo.FiltersDhcpOptions{
-			DhcpOptionsSetIds: &[]string{dhcpID.(string)},
-		}
-	}
-	if filtersOk {
+	if filters, filtersOk := d.GetOk("filter"); filtersOk {
 		params.Filters = buildOutscaleDataSourceDHCPOptionFilters(filters.(*schema.Set))
 	}
```

Comment:
This is a singular data_source, therefore to avoid matching with multiple data_source user should add
 ```
filtersilters, filtersOk := d.GetOk("filter")
if !filtersOk {
	return fmt.Errorf("filters must be assigned")
}
```